model/task/task.go

```diff
@@ -86,8 +86,8 @@ type Task struct {
 	TaskGroupOrder     int                 `bson:"task_group_order,omitempty" json:"task_group_order,omitempty"`
 	Logs               *apimodels.TaskLogs `bson:"logs,omitempty" json:"logs,omitempty"`
 	MustHaveResults    bool                `bson:"must_have_results,omitempty" json:"must_have_results,omitempty"`
-	HasCedarResults    bool                `bson:"has_cedar_results,omitempty" json:"has_cedar_results,omitempty"`
-	CedarResultsFailed bool                `bson:"cedar_results_failed,omitempty" json:"cedar_results_failed,omitempty"`
+	HasCedarResults    bool                `bson:"has_cedar_results" json:"has_cedar_results"`
+	CedarResultsFailed bool                `bson:"cedar_results_failed" json:"cedar_results_failed"`
```

Comment:
Is it necessary to change these to be non-omitempty in the DB? It seems like it would only have to be added to the REST model, which will still correctly see a false boolean in the case where it's omitted in the DB model.

Reply:
I guess it really doesn't matter, saves room in the DB so I will add them back.